src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -7,190 +7,190 @@ partial class ApplicationEngine
     {
         public static readonly IReadOnlyDictionary<OpCode, long> OpCodePrices = new Dictionary<OpCode, long>
```

Comment:
How about we make all prices become `1 << n` ?

Reply:
I'm not sure I get the idea. Some specific price coefficients can be adjusted for sure (based on https://github.com/neo-project/neo/issues/1875#issuecomment-709599810), providing more fairness, but I think `base_exec_fee * OpCodePrices[opcode]` scheme works well.

Reply:
> 1 << n 

I think the coefficient is too big, it's not as convenient as it is now

Reply:
I think opcodes with similar consumption should have the same price. We do not need and cannot make the price very precise. Because the consumption tested on different computers is also different. By fitting all price values to the form of `1 << n`, the price difference of similar orders can be eliminated.

Reply:
I'm not sure what it buys us. Of course no price table is perfect just because there is a huge difference between typical and worst cases for some opcodes. At the same time we have data for opcode execution times using two independent VM implementations and for many simple opcodes it gives very close relative numbers that can be used. Can these be rounded to powers of two? Sure they can, but we just lose some precision this way.

But let me try making a 2ⁿ rounding and testing it against preview3 transactions, maybe it's not that bad after all.

Reply:
See updated patch, the effect of it is somewhat similar to https://github.com/neo-project/neo/issues/1875#issuecomment-710540367, for real invocations a little GAS gets burned. I've tried again to reuse data we have, though every particular opcode can be discussed and changed if needed. We can of course store n in the table and not 2ⁿ, but that's not very convenient in some places. Tests deliberately left out for now.

Reply:
SYSCALL's price should be updated too.

Reply:
Do you mean `SYSCALL` opcode? IIRC setting it to non-zero value breaks things for some reason.

Reply:
I mean the `fixedCPUPrice` of all the SYSCALLs should be rounded to powers of two.

Reply:
OK, let's try that.

Reply:
Updated.